lib/common/util.js

```diff
@@ -38,11 +38,11 @@ var streamEvents = require('stream-events');
 var through = require('through2');
 var uniq = require('array-uniq');
 
-/** @const {object} gcloud-node's package.json file. */
-var PKG = require('../../package.json');
+/** @const {object} @google-cloud/common's package.json file. */
+var PKG = require('./package.json');
 
 /** @const {string} User agent. */
-var USER_AGENT = 'gcloud-node/' + PKG.version;
+var USER_AGENT = 'google-cloud-node/' + PKG.version;
```

Comment:
Is it useful to include the version of google-cloud/common in the user agent? Maybe we should expose a setter so each module can set a more useful header, i.e. `setUserAgent('@google-cloud/storage', STORAGE_PKG.version)`